README.md

```diff
@@ -1,2 +1,142 @@
+As such, it'll only watch for multus network annotations changes.
+
+### Removal
+Use `kubectl` to remove the controller from your cluster.
+
+```bash
+kubectl delete -f manifests/dynamic-networks-controller.yaml
+```
+
+### Adding / removing network interfaces
+To add (or remove...) network interfaces from a running pod, the user should
+simply edit the running pod network selection elements - i.e. the `k8s.v1.cni.cncf.io/networks`
+annotation.
+
+Thus, if we had this running pod (and `NetworkAttachmentDefinition`):
+```yaml
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: macvlan1-config
+spec:
+  config: '{
+            "cniVersion": "0.4.0",
+            "plugins": [
+                {
+                    "type": "macvlan",
+                    "capabilities": { "ips": true },
+                    "master": "eth1",
+                    "mode": "bridge",
+                    "ipam": {
+                        "type": "static"
+                    }
+                }, {
+                    "type": "tuning"
+                } ]
+        }'
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: macvlan1-worker1
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+            {
+                "name": "macvlan1-config",
+                "ips": [ "10.1.1.11/24" ]
+            }
+    ]'
+  labels:
+    app: macvlan
+spec:
+  containers:
+  - name: macvlan-worker1
+    image: docker.io/library/alpine:latest
+    command: ["/bin/sleep", "10000"]
+```
+
+We would run this example yaml to **add** an interface from it:
+```yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: macvlan1-worker1
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+            {
+                "name": "macvlan1-config",
+                "ips": [ "10.1.1.11/24" ]
+            },
+            {
+                "name": "macvlan1-config",
+                "interface": "ens4",
+            }
+    ]'
+  labels:
+    app: macvlan
+spec:
+  containers:
+    - name: macvlan-worker1
+      image: docker.io/library/alpine:latest
+      command: ["/bin/sleep", "10000"]
+```
+
+And we would run this example yaml to **remove** an interface to it:
+```yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: macvlan1-worker1
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[]' # this will remove all networks from the pod
+  labels:
+    app: macvlan
+spec:
+  containers:
+    - name: macvlan-worker1
+      image: docker.io/library/alpine:latest
+      command: ["/bin/sleep", "10000"]
+```
+
+## Configuration
+The `multus-dynamic-networks-controller` configuration is encoded in JSON, and allows the following keys:
+
+- `"criSocketPath"`: specify the path to the CRI socket. Defaults to `/run/containerd/containerd.sock`.
+- `"criType"`: either `crio` or `containerd`. Defaults to `containerd`.
+- `"multusSocketDir"`: directory where the multus socket will be found. Defaults to `/var/run/multus-cni/`.
+
```

Comment:
Please consider adding a paragraph explaining how to view the controller's logs.

Reply:
It's just a regular pod ... meaning you access logs via `kubectl logs -n<namespace> <pod name>` . 

Do you think that info is relevant ? It seems basic.

Reply:
I agree that it is basic.

> meaning you access logs via kubectl logs -n<namespace> <pod name>

Yeah, but what are they?
A user will have to look at the deployment manifest to get them.

Reply:
Can you rephrase ? I don't follow. 

Who's "they" ? The logs ?

Reply:
The names of the controller's pod, and the name of the namespace it is deployed to.

Reply:
It will obviously depend of which namespace you deploy the controller to, and what the ds name is. 

I still feel that info is pretty obvious to a user.

Reply:
Fair enough.